### ISSUE TITLE
Link does not work

### ISSUE THREAD

**Issue by vertesy** [nemoarchive/host-data-on-nemo#1]
"GUI interface software for BDBags

The software described [here](https://bd2k.ini.usc.edu/tools/bdbag/)"

Error:
We can’t connect to the server at bd2k.ini.usc.edu.

ps. It really doesn't help data accessibility that people cannot just download a .tar.gz etc file, but first need to learn this new awesome custom tool. I am sure it's amazing, I just would appreciate if I could spend that 30 minutes on something else.

I guess this is where I should get it from:
https://github.com/fair-research/bdbag
